recipe/build.sh

```diff
@@ -1,14 +1,18 @@
 #!/bin/sh
 
+cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
 mkdir build
 cd build
+export RUSTFLAGS="${CARGO_BUILD_RUSTFLAGS}"
+# CFLAGS -mcpu=power8 -mtune=power8 break the ppc64le build
+export CFLAGS=$(echo $CFLAGS | sed -E 's?-mcpu=[^ ]+ ??; s?-mtune=[^ ]+ ??')
```

Comment:
Given the comment, should this be ppc64le-only?

Reply:
Done in https://github.com/conda-forge/fish-feedstock/pull/49/commits/c3223feeb922744fac6fe5420690ebef85e4ac20